github-orgs/kubeflow/org.yaml

```diff
@@ -860,6 +861,7 @@ orgs:
             - richardsliu
             members:
             - Jeffwan
+            - jinchihe
```

Comment:
Please create a fairing release-team which has access to just the repos needed for fairing.

Reply:
Please remove yourself from this list as we are moving away from giving folks broad access and instead giving more narrowly scoped access.

Reply:
OK